sdks/java/io/hadoop/jdk1.8-tests/src/test/resources/kubernetes/cassandra/LargeITCluster/data-load.sh

```diff
@@ -0,0 +1,131 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Hashcode for 50m records is c89f996324e4dd4b6071a9b735281e0329a58b75
+
+# Script to load data using YCSB on Cassandra multi node cluster.
+ 
+#!/bin/bash
+
+set -e
+
+# Record count set to 50000000, change this value to load as per requirement.
+recordcount=50000000
+
+# Function to delete the temporary cassandra service in an erroneous and successful situation
+function delete_service {
+  cd ../LargeITCluster
+  kubectl delete -f cassandra-svc-temp.yaml
+}
+
+# Delete cassandra single node set up before exit 
+trap delete_service EXIT
+
+# Check if cassandra service already exists
+services_exists="$(kubectl get svc -o=name)"
+
+#Convert string of space(' ') separated words to array of words
+service_list=( $services_exists )
+
+for i in "${service_list[@]}"
+do
+   if [ "$i" == "services/cassandra-temp" ]; then
```

Comment:
I bet there's also a way to use "-l X" as is done on line 65, but to be honest, I'd rather just finish this PR 

(both these suggestions are non-blocking the PR - I'd rather not take a risky change to this and get it done by tomorrow)